src/main/webapp/translation.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1" charset="utf-8">
+    <title>EasyVote</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <header>
+      <nav class="navbar navbar-inverse">
+        <div class="container-fluid">
+          <div class="navbar-header">
+            <a class="navbar-brand" href="index.html">EasyVote</a>
+          </div>
+          <ul class="nav navbar-nav">
+            <li class="active"><a href="index.html">Home</a>
+            <li><a href="candidates.html">Candidate Info</a></li>
+            <li><a href="locations.html">Poll Locations</a></li>
+            <li><a href="translation.html">Translate</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <footer>
+    </footer>
+    <script src="script.js"></script>
```

Comment:
Everything looks good, but from what either Cam and Zu have said to me, it's best to put the script tags at the very end of the body. This is so the HTML content of the page can always load first